diff -r -c hugin/src/hugin1/icpfind/AutoCtrlPointCreator.cpp hugin-new/src/hugin1/icpfind/AutoCtrlPointCreator.cpp
*** hugin/src/hugin1/icpfind/AutoCtrlPointCreator.cpp	Sat Nov  3 19:12:42 2018
--- hugin-new/src/hugin1/icpfind/AutoCtrlPointCreator.cpp	Sat Nov  3 19:18:23 2018
***************
*** 85,91 ****
      else
      {
          wxString cmdline=prog+wxT(" ")+args;
!         return wxExecute(cmdline,wxEXEC_SYNC | wxEXEC_MAKE_GROUP_LEADER);
      };
  };
  
--- 85,97 ----
      else
      {
          wxString cmdline=prog+wxT(" ")+args;
!         wxArrayString output;
!         int ret = wxExecute(cmdline,output,wxEXEC_SYNC | wxEXEC_MAKE_GROUP_LEADER);
!         std::cout << cmdline << std::endl;
!         std::cout << "Output:" << std::endl;
!         for ( size_t n=0; n < output.GetCount(); ++n )
!           std::cout << output.Item(n) << std::endl;
!         return ret;
      };
  };
  
***************
*** 478,484 ****
  
      ret_value = 0;
      // use MyExternalCmdExecDialog
!     ret_value = CPExecute(autopanoExe, autopanoArgs, _("finding control points"), parent);
  
      if (ret_value == HUGIN_EXIT_CODE_CANCELLED) {
          return cps;
--- 484,490 ----
  
      ret_value = 0;
      // use MyExternalCmdExecDialog
!     ret_value = CPExecute(autopanoExe, autopanoArgs, _("finding control points"), NULL /*parent*/);
  
      if (ret_value == HUGIN_EXIT_CODE_CANCELLED) {
          return cps;
